src/lib/components/ui/Dropdown/menu/MenuButton.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" module>
+	export type Props = {
+		onClick?: () => void;
+		children?: any;
+		style?: String;
+	};
+</script>
+
+<script lang="ts">
+	let { onClick, children, style = 'bg-blue-500 text-white border-none ' }: Props = $props();
+</script>
+
+<button class="{style} py-2 px-4 cursor-pointer rounded-md" onclick={onClick}>
+	{@render children()}
```

Comment:
since children is optional (provided or not provided).
We must check if it is provided with {#if childre} ... {/if}